clang/test/Driver/unsupported-option.c

```diff
@@ -27,3 +27,10 @@
 // RUN: not %clang --target=x86_64 -### -mhtm -lc %s 2>&1 \
 // RUN:   | FileCheck %s -check-prefix=UNSUP_OPT
 // UNSUP_OPT: error: unsupported option
+
+
+// RUN: not %clang -c --target=aarch64-- -mfpu=crypto-neon-fp-armv8 %s 2>&1 | \
```

Comment:
This RUN line without -Qunused-arguments doesn't add test coverage. It can be removed